components/places/src/history_sync/engine.rs

```diff
@@ -141,3 +141,8 @@ impl SyncEngine for HistorySyncEngine {
         Ok(())
     }
 }
+
+pub fn set_sync_ids(conn: &PlacesDb, global: String, collection_id: String) -> Result<()> {
```

Comment:
nit: a comment here or in the UDL that this is only used by iOS migration and should be killed as part of killing that might be helpful to our future selvss.